src/bosh-director/lib/bosh/director/jobs/update_stemcell.rb

```diff
@@ -81,41 +82,50 @@ def perform
         stemcell = nil
         cloud_factory.all_configured_clouds.each do |cloud|
           cpi_suffix = " (cpi: #{cloud[:name]})" unless cloud[:name].blank?
-
+          supported = true
```

Comment:
I just didn't want to do extra checks (+ cpi call) in case if it is not necessary and stemcell is already available. But I see your point and will update the code